src/Plugin/GraphQL/Schema/ThunderSchema.php

```diff
@@ -61,43 +61,61 @@ public function getResolverRegistry() {
   }
 
   /**
-   * TODO: this should be much less hard coded and better pluggable.
+   * Function addInterfaces.
    *
    */
   protected function addInterfaces () {
     $this->registry->addTypeResolver('ContentType', function ($value) {
+      $bundle = $value->bundle();
+
       if ($value instanceof ContentEntityInterface) {
-        return ucfirst($value->bundle());
+        return $this->mapBundleToSchemaName();
       }
-      throw new Error('Could not resolve content type.');
+      throw new Error('Could not resolve content type.' . $bundle);
```

Comment:
Please add space after the colon for better readability

Reply:
ok